packages/relay/src/lib/constants.ts

```diff
@@ -236,6 +236,7 @@ export default {
   },
 
   MAX_TRANSACTION_FEE_THRESHOLD: ConfigService.get('MAX_TRANSACTION_FEE_THRESHOLD'),
+  MAX_GAS_ALLOWANCE_HBAR: ConfigService.get('MAX_GAS_ALLOWANCE_HBAR'),
```

Comment:
Q: why do we add it here when we use it straight from the ConfigService everywhere via `ConfigService.get('MAX_GAS_ALLOWANCE_HBAR')`?

Reply:
I've noticed that other constant parameters (e.g. `GAS_PRICE_TINY_BAR_BUFFER`) are used from both the constants object and the current live env values. I therefore assumed that this was the best practice to save the startup value in case someone wants to access it instead of the current value.

As you correctly noted, at the moment `MAX_GAS_ALLOWANCE_HBAR` is never read from the constants object, so if you prefer I can remove it from there.

Reply:
Hi @natanasow I removed `MAX_GAS_ALLOWANCE_HBAR` from the constants because at the moment it is only used via the ConfigService.